docs/test_runners.md

```diff
@@ -0,0 +1,42 @@
+# Elección de `test runner`
+> Objetivo 4
+
+## Criterios
+Es de principal importancia que la decisión se tome teniendo en cuenta los principios F.I.R.S.T.
```

Comment:
No entiendo por qué pensáis que listar los principios aquí, sin hacer ninguna referencia a dónde lo habéis usado efectivamente, sirve de algo. ¿Has comprobado que son fast? ¿Cómo sé que se validan a sí mismos?

Reply:
:arrow_up: